docs/install.rst

```diff
@@ -184,6 +184,30 @@ globally.
    available globally. This helps avoid issues caused by hiding the public
    schema from queries.
 
+Working with Tenant specific schemas
+====================================
+Since each Tenant has it's own database you need a way to tell Django what
+database to use when using the management commands. 
+
+A special management command ``tenant_command`` has been added to allow you to
+execute Django management commands on a specific Tenant schema.
+
+.. code-block:: python
+
+    python manage.py tenant_command loaddata --schema=my_tenant test_fixture
+
+Creating a new Tenant
```

Comment:
This is already documented in `use.rst` – I suggest cross-reference here rather than re-documenting.